util/MySqlMigrations/Migrations/20231214162533_GrantIdWithIndexes.cs

```diff
@@ -78,7 +78,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
             table: "Grant",
             type: "int",
             nullable: false,
-            defaultValue: 0)
+            defaultValue: 1)
```

Comment:
ℹ️ Override for what's generated (0). 0 is not allowed here. Same goes for the other two.

Reply:
I don't think defaultvalue as a line is needed, the identity will always provide value

Reply:
`migrationBuilder.AddColumn<int>(
    name: "Id",
    table: "Grant",
    type: "int",
    nullable: false)
    .Annotation("MySql:ValueGenerationStrategy", MySqlValueGenerationStrategy.IdentityColumn);
`